### PR TITLE
🐛Add date-template and info-template to amp-date-picker validation.

### PR DESCRIPTION
**Pull request by cvializ**
These were missing, making the date picker example on amp-by-example invalid.

### CODE REVIEW

File: extensions/amp-date-picker/validator-amp-date-picker.protoascii [hunk 1]
```diff
@@ -191,3 +191,69 @@ attr_lists: {
     value: ""
   }
 }
+
+# HTML5, 4.11.3 The template element
```
Comment (honeybadgerdontcare) [inline extensions/amp-date-picker/validator-amp-date-picker.protoascii]: The `template` tag is defined in the `amp-mustache` extension as well. Please add a comment here noting that and also there noting these are defined here. See https://github.com/ampproject/amphtml/blob/master/extensions/amp-mustache/validator-amp-mustache.protoascii

File: extensions/amp-date-picker/validator-amp-date-picker.protoascii [hunk 2]
```diff
@@ -191,3 +191,69 @@ attr_lists: {
...
+tags: {
+  html_format: AMP
+  tag_name: "TEMPLATE"
+  spec_name: "amp-date-picker > template [date-template] [dates]"
+  mandatory_parent: "AMP-DATE-PICKER"
+  requires_extension: "amp-mustache"
+  attrs: {
```
Comment (honeybadgerdontcare) [inline extensions/amp-date-picker/validator-amp-date-picker.protoascii]: Please alphabetize by attribute name (here and below).

File: extensions/amp-date-picker/validator-amp-date-picker.protoascii [hunk 3]
```diff
@@ -191,3 +191,69 @@ attr_lists: {
...
+  }
+  attrs: {
+    name: "default"
+    mandatory: true
+    value: ""
+    dispatch_key: NAME_DISPATCH
+  }
+}
+
+# HTML5, 4.11.3 The template element
```
Comment (honeybadgerdontcare) [inline extensions/amp-date-picker/validator-amp-date-picker.protoascii]: no need to repeat this comment or have blank lines between `tags: {}`

**Comment by cvializ** [comment]
Thanks for the review! PTAL

**Pull request merged**